CHANGELOG.rst

```diff
@@ -12,7 +12,11 @@ Change Log
 .. There should always be an "Unreleased" section for changes pending release.
 
 Unreleased
-~~~~~~~~~~
```

Comment:
Nice catch!

Reply:
Oh, I didn't mean that, I just meant that yoiu updated the changelog at all lmso